getGeocode.js

```diff
@@ -0,0 +1,80 @@
+const dotenv = require("dotenv");
+//note: this only works locally; in Lambda we use environment variables set manually
+dotenv.config();
+
+const fs = require("fs");
+const fetch = require("node-fetch");
+
+const { Client } = require("@googlemaps/google-maps-services-js");
+const geo = require("./geo.json");
+
+const getGeocode = async (name, street, zip) => {
+  const address = `${name},MA,${street},${zip}`;
+  const client = new Client();
+  try {
+    const resp = await client.geocode({
+      params: {
+        address,
+        key: process.env.GOOGLE_API_KEY,
+      },
+    });
+    return resp.data;
+  } catch (e) {
+    console.error(e.response.data);
+  }
+};
+
+const getLocationData = async () => {
+  const res = await fetch(
+    "https://mzqsa4noec.execute-api.us-east-1.amazonaws.com/prod"
+  );
+  const data = await res.json();
+
+  let locationLookup = {};
+  for (const location of JSON.parse(data.body).results) {
+    const {
+      name = "",
+      street = "",
+      zip = "",
+      resolvedLocation,
+      latitude,
+      longitude,
+    } = location;
+
+    const locationInd = `${name}|${street}|${zip}`;
```

Comment:
I suggest looking at the file data/dataDefaulter.js and using some of the code (not copy-paste, but actually using it/refactoring it out)